Makefile

```diff
@@ -418,6 +424,11 @@ $(ALERTMANAGER):
 $(MINIO_SERVER):
 	$(call fetch_go_bin_version,github.com/minio/minio,$(MINIO_SERVER_VERSION))
 
+$(FAILLINT):
+	@go get github.com/fatih/faillint@$(FAILLINT_VERSION)
+	@mv $(GOBIN)/faillint $(FAILLINT)
+	@go mod tidy
```

Comment:
Maybe the problem is that the go mod file is changed because of downloading faillint. What about using `fetch_go_bin_version` like we downloading other binaries?

Reply:
Fixed